angular/src/components/login.component.ts

```diff
@@ -61,6 +66,20 @@ export class LoginComponent implements OnInit {
         if (Utils.isBrowser && !Utils.isNode) {
             this.focusInput();
         }
+
+        let webVaultUrl = this.environmentService.getWebVaultUrl();
```

Comment:
Please see updates from https://github.com/bitwarden/jslib/pull/435

Reply:
Turns out this was a function of me branching off of an old commit.